src/Connection/Replication/SentinelReplication.php

```diff
@@ -538,7 +539,15 @@ protected function assertConnectionRole(NodeConnectionInterface $connection, $ro
     {
         $role = strtolower($role);
         $actualRole = $connection->executeCommand(RawCommand::create('ROLE'));
-
+        if (is_object($actualRole) && $actualRole instanceof Error) {
```

Comment:
No need for `is_object()` here, you can just check if `$actualRole` is an instance of `Predis\Response\ErrorInterface` (instead of the actual `Predis\Response\Error` class).